### ISSUE TITLE
Review and fix one step dissolution modeling

### ISSUE THREAD

**Issue by martinamorris** [EpiModel/EpiModel#549]
Taking off from #525, it seems like there may be different options to consider for simulating these onetime partnership networks in `tergm`.  @krivit referenced [this issue in the tergm repo](https://github.com/statnet/tergm/issues/41) and pointed out that

````
simulate(nw ~ edges, dynamic=TRUE, time.slices=100, coef=theta, output="networkDynamic")
````
will generate a `networkDynamic` object each of whose 100 time steps is governed by whatever model/theta are specified.  Note this relies on the `tergm::simulate_formula.network()` function.

**Comment by smjenness** [comment]
@ThomasKraft : No, you should be all set. All of the above is more related to consistent in network storage for simulations across different parameterizations. There was one issue related to time varying attributes that is indeed a bug (my comment directly above) but that only concerns ndtv network movies. Fingers crossed for your paper!

**Comment by chad-klumb** [comment]
@smjenness Are you opposed to handling the "ergm" (duration = 1) case in the same way the "tergm" (duration > 1) case is currently handled, in the sense of how the networkDynamic is generated during the simulation, and using tergm controls (rather than ergm controls) to modify the simulation when needed?

**Comment by chad-klumb** [comment]
(i.e., generating independent cross-sectional networks at successive time points when duration = 1, but recording e.g. edge activity in the networkDynamic following the conventions of the tergm package's simulate functions, and using control arguments for those functions)

**Comment by chad-klumb** [comment]
implemented with tergm simulation in #648

if need be this can be changed to ergm simulation, but since netsim defines a time step, it is not entirely unreasonable to use tergm simulation even if the model is cross-sectional
